### ISSUE TITLE
Crash on World Load, migration attempt.

### ISSUE THREAD

**Issue by PiggyWhiskey** [dewiniaid/BlueprintExtensions#8]
![image](https://user-images.githubusercontent.com/9373995/55313653-7a525b00-54a7-11e9-9a19-682834604af3.png)

```
 326.515 Applying migration: Blueprint Extensions: 2019-03-27-v0_4_0_gui_changes_part_deux.lua
 327.119 Error AppManagerStates.cpp:1397: Error while applying migration: Blueprint Extensions: 2019-03-27-v0_4_0_gui_changes_part_deux.lua

..._/migrations/2019-03-27-v0_4_0_gui_changes_part_deux.lua:2: module GUI not found;  no such file __BlueprintExtensions__/migrations/GUI.lua no such file __BlueprintExtensions__/GUI.lua no such file D:/Factorio/FactorioGame/data/core/lualib/GUI.lua
stack traceback:
	[C]: in function 'require'
	..._/migrations/2019-03-27-v0_4_0_gui_changes_part_deux.lua:2: in main chunk
```

**Comment by Ristellise** [comment]
Still happens with 0.4.1

**Comment by Omnifarious** [comment]
In fact, it happens now with 0.4.1 when it didn't happen before for me:

![BPE-Error](https://user-images.githubusercontent.com/597857/55333533-5e6d9b80-544c-11e9-93e1-dfd1148f491f.png)

**Comment by H4nSolo** [comment]
GUI.lua is missing in the Factorio Core!!!!

**Comment by dewiniaid** [comment]
Fix for this is incoming.

The file is actually `gui.lua` (lowercase), but Factorio doesn't complain when unzipped so I didn't catch it during testing.

**Comment by dewiniaid** [comment]
Should now be fixed in the now available 0.4.2.

Not entirely certain how the fix for 0.4.1 didn't end up making it in the release zip.

**Comment by Omnifarious** [comment]
It doesn't complain when unzipped because of Windows. I really dislike case insensitive filesystems.
